code/game/objects/structures/crates_lockers/closets/secure/security.dm

```diff
@@ -285,7 +284,6 @@
 	new /obj/item/clothing/under/lawyer/oldman(src)
 	new /obj/item/clothing/under/lawyer/black(src)
 	new /obj/item/clothing/under/lawyer/female(src)
-	new /obj/item/clothing/under/rank/centcom/representative(src)

```

Comment:
This is the representative CC outfit. Since they are a CC VIP this makes sense, since its the **representative** outfit, not the naval outfit

Reply:
If you look at the actual description, it says navy.

Reply:
Why not change the descriptions then, instead of just deleting them all?

Reply:
@TDSSS Because giving people a different typepath to what is virtually the same uniform as navy officers makes no sense, when they aren't navy officers. It uses the same sprites and everything. They have their own uniforms already.